aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Choose an ASP.NET Core UI
+author: wadepickett
+description: Understand when to use the various ASP.NET Core web UI technologies Microsoft provides and supports.
+ms.author: wpickett
+ms.date: 06/30/2021
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: tutorials/choose-web-ui
+---
+# Choose an ASP.NET Core web UI
+
+ASP.NET Core is a complete end to end UI framework.  You choose which functionalities to combine that fit your web UI app needs.
+
+Start by determining if your web UI app will primarily benefit from pre-rendering UI on the server, or rendering UI at the client browser. Then review the server rendered or client rendered ASP.NET Core project types to find which best fit your apps needs. 
+
+## Choose an ASP.NET Core web UI technology
+
+Use the following flowchart to select an ASP.NET Core web UI technology:
+
+![Decision tree for ASP.NET Core web UI technology](choose-web-ui/_static/choose_ui.png)
+
+## Choose server rendered UI or client rendered UI
+
+There are two general approaches to building modern web UI:
+
+* Apps that render UI on the server.
+* Apps that render UI on the client in the browser.
+
+There are benefits and drawbacks to consider for either approach. A hybrid approach is also possible.  For example, most of the web UI is rendered on the server, and client rendered pages are added as needed.
+
+### When to choose a server rendered UI approach
+
+A web UI app that renders on the server dynamically generates the page's HTML and CSS on the server. The page arrives at the client ready to display.
+
+Benefits:
+
+* The client requirements are minimal since the server does the work of logic and page generation:
+  * Great for low-end devices and low-bandwidth connections.
+  * Allows for a broad range of browser versions at the client.
+  * Minimal to no code to pull to the client.
+* Flexibility of access to server resources, in a protected environment. For example:
+  * Database access.
+  * Access to secrets, such as an access key used for API calls to Azure storage.
+* Uses a token-based security model, rather than using cookies.
+
+Examples of server rendered web UI apps:
```

Comment:
I'm don't think we can say that these app types are always or even typically server rendered. These app types can be implemented using a variety of approaches.